server/issue.go

```diff
@@ -111,7 +111,7 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 			ChannelId: post.ChannelId,
 			RootId:    rootId,
 			ParentId:  parentId,
-			UserId:    mattermostUserId,
+			UserId:    ji.GetPlugin().conf.botUserID,
```

Comment:
@levb Correct method to retrieve UserId for Plugin?

Reply:
should use p.getConfig()

Reply:
No. You need to use the `getConfig` method.